rpc/block_svc_routines.c

```diff
@@ -4375,6 +4377,23 @@ block_create_common(blockCreate *blk, char *control, char *volServer, char *prio
   }
   reply->exit = -1;
 
+  if (GB_ALLOC_N(reply->out, 8192) < 0) {
+    goto out;
+  }
+
+  if (GB_ASPRINTF(&exec, "grep -s 'name.*%s' /etc/target/saveconfig.json /etc/target/saveconfig.json.temp| wc -l",
```

Comment:
@lxbsz 

-> use grep -c instead of  '| wc -l'
-> I didn't fully understand why are we checking for 'name.*<block-name>' here ?
-> Should we check more strictly ? like ---> ``` \"name\"\:\ \"<block-name>\"\, ``` <--- ? along with grep -w option ?

Reply:
@pkalever 
I am okay with this, here using the name just want to filter the string from the StorageObjects.

The "name.*<block-name>" is already enough in this case. And just also want to make the code more flexible in case in future there has any change of "\"name\"\:\ \"<block-name>\"". Here it is one whitespace after ':', will be possible will be more in future or for different json-c verison ?

Reply:
@pkalever 
If this won't okay for you, then I could fix it :-)

Thanks,